config/initializers/session_store.rb

```diff
@@ -1,8 +1,3 @@
 # Be sure to restart your server when you modify this file.
 
-Elovation::Application.config.session_store :cookie_store, key: '_elovation_session'
-
```

Comment:
I've noticed site wide that we're dropping the Elovation namespace, does this have something to do with Rails 5? Drew made it this way, but never specified why do you have any insight on why he may have decided to do it this way?